bitnami/thanos/templates/query/service-grpc-headless.yaml

```
@@ -0,0 +1,37 @@
+{{- if and .Values.query.enabled .Values.query.serviceGrpc.additionalHeadless }}
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ include "common.names.fullname" . }}-query-grpc-headless
+  namespace: {{ .Release.Namespace | quote }}
+  labels: {{- include "common.labels.standard" . | nindent 4 }}
+    app.kubernetes.io/component: query
+    {{- if .Values.commonLabels }}
+    {{- include "common.tplvalues.render" ( dict "value" .Values.commonLabels "context" $ ) | nindent 4 }}
+    {{- end }}
+  annotations:
+    {{- if .Values.commonAnnotations }}
+    {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
+    {{- end }}
+    {{- if .Values.query.serviceGrpc.annotations }}
+    {{- include "common.tplvalues.render" (dict "value" .Values.query.serviceGrpc.annotations "context" $) | nindent 4 }}
+    {{- end }}
+spec:
+  type: ClusterIP
+  clusterIP: None
+  ports:
+    - port: {{ if .Values.query.service.grpc }}{{ coalesce .Values.query.service.ports.grpc .Values.query.service.grpc.port }}{{ else }}{{ .Values.query.service.ports.grpc }}{{ end }}
```

Typo here and I think we can simplify:
```suggestion
    - port: {{ .Values.query.serviceGrpc.ports.grpc }}
```